data/sql/updates/pending_db_world/rev_1643395587559675400.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643395587559675400');
+
+CREATE TABLE IF NOT EXISTS `instance_saved_data` (
+  `instance_id` int unsigned NOT NULL,
+  `gameobject_entry` int unsigned NOT NULL,
+  `gameobject_guid` int unsigned NOT NULL,
+  `gameobject_state` int unsigned DEFAULT '0',
```

Comment:
I'd like to say id, entry, guid, state

Reply:
that seems better yep, its simpler, we can keep it simple